pkg/common/malloc/class_allocator.go

```diff
@@ -100,16 +106,24 @@ func newFixedSizedAllocator(size uint64) *fixedSizeMmapAllocator {
 	// objects in buffer2 will be MADV_DONTNEED-advised and will not occupy RSS, so it's safe to use a large number
 	num2 := 1024
 	ret := &fixedSizeMmapAllocator{
-		size:    size,
-		buffer1: make(chan unsafe.Pointer, num1),
-		buffer2: make(chan unsafe.Pointer, num2),
+		size:          size,
+		checkFraction: checkFraction,
+		buffer1:       make(chan unsafe.Pointer, num1),
+		buffer2:       make(chan unsafe.Pointer, num2),
 	}
 	return ret
 }
 
 var _ Allocator = new(fixedSizeMmapAllocator)
 
-func (f *fixedSizeMmapAllocator) Allocate(_ uint64) (unsafe.Pointer, Deallocator) {
+func (f *fixedSizeMmapAllocator) Allocate(_ uint64) (ptr unsafe.Pointer, dec Deallocator) {
+	defer func() {
+		n := fastrand()
+		if n%f.checkFraction == 0 {
+			dec = newCheckedDeallocator(dec)
```

Comment:
use f.checkFraction as 0 to disable this checked allocator/deallocator.

Reply:
done